kclvm/api/src/testdata/get-schema-type-mapping.json

```diff
@@ -1,4 +1,4 @@
 {
 	"file": "schema.k",
-	"code": "@info(name=\"ServerSchema\")\nschema Server:\n    \"\"\"Server is the common user interface for long-running\n    services adopting the best practice of Kubernetes.\n\n    Attributes\n    ----------\n    workloadType: str, default is \"Deployment\", required\n        Use this attribute to specify which kind of long-running service you want.\n        Valid values: Deployment, CafeDeployment.\n        See also: kusion_models/core/v1/workload_metadata.k.\n    name: str, required\n        A Server-level attribute.\n        The name of the long-running service.\n        See also: kusion_models/core/v1/metadata.k.\n    labels: {str:str}, optional\n        A Server-level attribute.\n        The labels of the long-running service.\n        See also: kusion_models/core/v1/metadata.k.\n\n    Examples\n    ----------------------\n    myCustomApp = AppConfiguration {\n        name = \"componentName\"\n    }\n    \"\"\"\n    workloadType: str = \"Deployment\"\n    @info(\"name\", key=\"value\")\n    name: str\n    labels?: {str:str}\n    useCustomizeLables: True | False\n    containers: [Container]\n\nschema Container:\n    \"\"\"Container is the common user interface for long-running services.\n\n    Attributes\n    ----------\n    name: str, required\n        The name of the long-running container.\n    \"\"\"\n    name: str\n"
```

Comment:
Pls add more description about this PR and run `make fmt` to format the whole source code.

Besides, U can change the API protobuf definition directly about the `example` field. When the compilation and testing are completed, the structure on the rust side will automatically change.

Reply:
- code re-formatted
- PR description updated
ptal~